### PR TITLE
Fix index alignment of pypsa loads time series

### PR DESCRIPTION
**Pull request by FabianHofmann**
Use `get_switchable_as_dense` to align timeseries index with static index.

@jenhagg could you try this out?

### CODE REVIEW

File: powersimdata/input/converter/pypsa_to_profiles.py [hunk 1]
```diff
@@ -67,9 +67,9 @@ def get_pypsa_demand_profile(network):
         raise TypeError("network must be a Network object")
 
     if not network.loads_t.p.empty:
-        demand = network.loads_t.p.copy()
+        demand = network.get_switchable_as_dense("Load", "p_set")
     else:
-        demand = network.loads_t.p_set.copy()
+        demand = network.get_switchable_as_dense("Load", "p_set")
```
Comment (BainanXia) [inline powersimdata/input/converter/pypsa_to_profiles.py]: It seems we are doing the same thing in both `if` and `else` branches?

**Comment by jenhagg** [comment]
This does seem to fix the issue (we now have `5963 in demand.columns`).

File: powersimdata/input/converter/pypsa_to_profiles.py [hunk 2]
```diff
@@ -67,9 +67,9 @@ def get_pypsa_demand_profile(network):
         raise TypeError("network must be a Network object")
 
     if not network.loads_t.p.empty:
-        demand = network.loads_t.p.copy()
+        demand = network.get_switchable_as_dense("Load", "p_set")
```
Comment (FabianHofmann) [inline powersimdata/input/converter/pypsa_to_profiles.py]:
```suggestion
        demand = network.loads_t.p.copy()
```

**Review by rouille — APPROVED** [review]
🙏

**Review by jenhagg — APPROVED** [review]

**Review by BainanXia — APPROVED** [review]

**Pull request merged**